.github/workflows/build.yml

```diff
@@ -66,3 +66,11 @@ jobs:
         env:
           ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: ${{ secrets.SONATYPE_NEXUS_USERNAME }}
           ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
+
+      - name: Deploy docs to website
+        uses: JamesIves/github-pages-deploy-action@releases/v3
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: gh-pages
+          FOLDER: build/dokka/html
```

Comment:
I suppose `uploadArchives` creates this folder?

Reply:
hmm, doesn't look like it.  I just cribbed this from `https://github.com/cashapp/turbine/blob/trunk/.github/workflows/build.yaml#L50`

Reply:
Ah, my guess is that, in build.yaml, either `build` or `publish` (prob the former) executes the dokka task which creates that folder?

Reply:
And in release.yaml, `dokkaHtml` does it?  https://github.com/cashapp/turbine/blob/trunk/.github/workflows/release.yaml#L64-L65

Reply:
hmm, i should also update the dokka plugin...

Reply:
Ok, had to re-learn the dokka config 😅 

This repo doesn't use the html style that Turbine uses, but rather the gfm style that SqlDelight et al. use.